apps/proxy/dynamic_config.yml

```diff
@@ -25,6 +25,10 @@ http:
       rule: "PathPrefix(`/api/search`)"
       service: search
       priority: 200
+    to-charity:
+      rule: "PathPrefix(`/api/charity`)"
+      service: charity
+      priority: 200
```

Comment:
You are setting the path to be `/api/charity` yet it seems your endpoints defined in `index.ts` are going to `/api/charities`. Which is it?